crates/typed-store/src/rocks/mod.rs

```diff
@@ -388,7 +380,6 @@ where
     }
 
     /// Convenience method for batch removal
-    #[instrument(level = "debug", skip_all, err)]
```

Comment:
Instead of removing these we could just lower the level to trace

Reply:
I agree with @bmwill .... it's useful to instrument key calls, but probably not at debug (but there are lots of things I want to downgrade from debug to trace)

Reply:
done